### PR TITLE
feat(cascading): avoid label setting on cascading pull request

### PR DESCRIPTION
**Pull request by kpanot**
## Proposed change

Avoid label setting on cascading pull request.

### CODE REVIEW

**Review by vscaiceanu-1a — APPROVED** [review]

File: .github/workflows/pr-labels.yml [hunk 1]
```diff
@@ -28,9 +28,25 @@ jobs:
         with:
           script: |
             const [owner, repo] = '${{ github.repository }}'.split('/');
-            github.rest.issues.addLabels({
+            const pLabels = github.rest.issues.listLabelsOnIssue({
               owner,
               repo,
               issue_number: context.issue.number,
               labels: ${{ env.LABELS }}
```
Comment (matthieu-crouzet) [inline .github/workflows/pr-labels.yml:35]: should you still give the param `labels` to retrieve the list ?

Comment (kpanot) [reply .github/workflows/pr-labels.yml:35]: It's not possible to have params/input to a PR triggered workflow but I can move it out to `env` to be more easily configurable

File: .github/workflows/pr-labels.yml [hunk 2]
```diff
@@ -28,9 +29,25 @@ jobs:
...
```
Comment (matthieu-crouzet) [inline .github/workflows/pr-labels.yml:36]:
```suggestion
            const pLabels = github.rest.issues.listLabelsOnIssue({
              owner,
              repo,
              issue_number: context.issue.number
```

Comment (kpanot) [reply .github/workflows/pr-labels.yml:36]: indeed, copy-pasta

**Review by matthieu-crouzet — APPROVED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Pull request merged**